theano/gpuarray/elemwise.py

```diff
@@ -2493,6 +2493,13 @@ def gpu_kernels(self, node, nodename):
         return kernels
 
 
+class GpuGammaLn(GammaLn):
+
+    def c_headers(self):
+        return ['math_functions.h']
```

Comment:
I don't think this helps anything, because the header will be for the op code, not the kernel.

Also, we can't add headers to kernels since nvrtc needs special parameters for them.